src/storage/storage.h

```diff
@@ -123,7 +123,10 @@ class Storage {
   Status SetOptionForAllColumnFamilies(const std::string &key, const std::string &value);
   Status SetDBOption(const std::string &key, const std::string &value);
   Status CreateColumnFamilies(const rocksdb::Options &options);
-  Status CreateBackup();
+  // The sequence_number will be pointed to the value of the sequence number in range of DB,
+  // but can't promise it's the latest sequence number. So you must check it by yourself before
+  // using it.
+  Status CreateBackup(uint64_t *sequence_number = nullptr);
```

Comment:
Where will we use it? since kvrocks is currently not a library but a execuable.

Reply:
As mentioned in https://github.com/apache/kvrocks/discussions/1986, they would like to implement the increment backup based on the codebase, so need to export the backup sequence number for this purpose.

Reply:
I think if they have such a requirement, maybe they can implement it in their codebase rather than modifying our codebase.

Reply:
Yes, I consider adding this because it won't add the extra maintenance burden for the codebase. And it's a common requirement to let them know where's the sequence number of the current backup.

Reply:
I see.